doc/code_snippets/snippets/mysql/instances.enabled/config.yaml

```diff
@@ -0,0 +1,25 @@
+app:
+  file: 'example.lua'
+
+groups:
+  group001:
+    replicasets:
+      replicaset001:
+        instances:
+          instance001:
+            iproto:
+              listen: "3301"
+            memtx:
+              memory: 128 * 1024 * 1024
+              min_tuple_size: 16
+              max_tuple_size: 128 * 1024 * 1024
+            wal:
+              mode: "none"
+              wal_max_size: 56 * 1024 * 1024
+            snapshot:
+              by_interval: 60 * 60
+              count: 6
+            log:
+              level: 7
+            fiber:
+              too_long_threshold: 0.5
```

Comment:
I think that all these option are not related to the article. How about removing them?

Reply:
I agree with it, seems like a good suggestion. I'll remove these options.